Python/ast.c

```diff
@@ -5020,7 +5020,7 @@ fstring_find_expr(const char **str, const char *end, int raw, int recurse_lvl,
     int conversion = -1; /* The conversion char.  Use default if not
                             specified, or !r if using = and no format
                             spec. */
-    int equal_conversion = 0; /* Are we using the = conversion? */
+    int equal_conversion = 0; /* Are we using the = feature? */
```

Comment:
So you still call the variable "equal_conversion"?  I figured you'd rename it.  "equal_debugging" maybe?